lib/node/runner/checker_ballot_transaction.go

```diff
@@ -175,18 +175,20 @@ func ValidateTx(st *storage.LevelDBBackend, tx transaction.Transaction) (err err
 	}
 
 	for _, op := range tx.B.Operations {
-		if err = ValidateOp(st, op); err != nil {
+		if err = ValidateOp(st, ba, op); err != nil {
 			return
 		}
 	}
 
 	return
 }
 
-func ValidateOp(st *storage.LevelDBBackend, op transaction.Operation) (err error) {
+func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op transaction.Operation) (err error) {
 	switch op.H.Type {
 	case transaction.OperationCreateAccount:
-		if _, ok := op.B.(transaction.OperationBodyCreateAccount); !ok {
+		var ok bool
+		var casted transaction.OperationBodyCreateAccount
```

Comment:
I hope that `casted` renamed to frozen.

Reply:
Why ? The fact that it is casted does not mean that its frozen. It could be a regular account creation.

Reply:
ah. sorry, i misunderstood.